Move job abandonment section (#1375)
mattermost/mattermost-handbook: operations/workplace/people/people-policies.md

Mattermost's Affirmative Action Plan can be found [here](https://docs.google.com/document/d/15PVfR5uI7TaWuSznLr3hmQd_4aAE-OR3AixVPEO-QF4/edit#heading=h.cbr53qvhhkxh).

## Job abandonment

> When a staff member is absent from work for three consecutive workdays and there is no entry on the [Time Away Calendar](https://calendar.google.com/calendar/embed?src=mattermost.com_mg3nulgf6e70e50ohlq2rrkcnc%40group.calendar.google.com&ctz=America%2FLos_Angeles) or a post in the [Stand-Up Channel](https://community.mattermost.com/private-core/channels/stand-up) noting the time off, and fails to contact their manager or HR, they can be terminated for job abandonment unless otherwise required by local law.

Reviewer: ```suggestion
When a staff member is absent from work for 3 consecutive workdays, and there is no entry on the [Time Away Calendar](https://www.google.com/url?q=http://app.rippling.com/api/feed/calendar/pto/company/are2zz48b0jkynqd/b91837056f386ea9355b4e71f456c24a381efea19840116f0779e81f053199e4/calendar.ics?company%3D658390ff2cae6bf84a79bcb5&sa=D&source=docs&ust=1718123593252322&usg=AOvVaw0l7BJEx680vP_7GJ520jMs), or a post in the [Stand-Up Channel](https://hub.mattermost.com/private-core/channels/stand-up) noting the time off, and the staff member fails to contact their manager or HR, they can be terminated for job abandonment unless otherwise required by local law.
```

> When a staff member is absent from work for 3 consecutive workdays, and there is no entry on the [Time Away Calendar](http://app.rippling.com/api/feed/calendar/pto/company/are2zz48b0jkynqd/b91837056f386ea9355b4e71f456c24a381efea19840116f0779e81f053199e4/calendar.ics?company%3D658390ff2cae6bf84a79bcb5&sa=D&source=docs&ust=1718123593252322&usg=AOvVaw0l7BJEx680vP_7GJ520jMs), or a post in the [Stand-Up Channel](https://hub.mattermost.com/private-core/channels/stand-up) noting the time off, and the staff member fails to contact their manager or HR, they can be terminated for job abandonment unless otherwise required by local law.

Author: ```suggestion
When a staff member is absent from work for 3 consecutive workdays, and there is no entry on the [Who's Out Calendar](http://app.rippling.com/api/feed/calendar/pto/company/are2zz48b0jkynqd/b91837056f386ea9355b4e71f456c24a381efea19840116f0779e81f053199e4/calendar.ics?company=658390ff2cae6bf84a79bcb5), or a post in the [Stand-Up Channel](https://hub.mattermost.com/private-core/channels/stand-up) noting the time off, and the staff member fails to contact their manager or HR, they can be terminated for job abandonment unless otherwise required by local law.
```

> When a staff member is absent from work for 3 consecutive workdays, and there is no entry on the [Time Away Calendar](https://www.google.com/url?q=http://app.rippling.com/api/feed/calendar/pto/company/are2zz48b0jkynqd/b91837056f386ea9355b4e71f456c24a381efea19840116f0779e81f053199e4/calendar.ics?company%3D658390ff2cae6bf84a79bcb5&sa=D&source=docs&ust=1718123593252322&usg=AOvVaw0l7BJEx680vP_7GJ520jMs), or a post in the [Stand-Up Channel](https://hub.mattermost.com/private-core/channels/stand-up) noting the time off, and the staff member fails to contact their manager or HR, they can be terminated for job abandonment unless otherwise required by local law.

Reviewer: ```suggestion
When a staff member is absent from work for 3 consecutive workdays, and there is no entry on the [Time Away Calendar](http://app.rippling.com/api/feed/calendar/pto/company/are2zz48b0jkynqd/b91837056f386ea9355b4e71f456c24a381efea19840116f0779e81f053199e4/calendar.ics?company%3D658390ff2cae6bf84a79bcb5&sa=D&source=docs&ust=1718123593252322&usg=AOvVaw0l7BJEx680vP_7GJ520jMs), or a post in the [Stand-Up Channel](https://hub.mattermost.com/private-core/channels/stand-up) noting the time off, and the staff member fails to contact their manager or HR, they can be terminated for job abandonment unless otherwise required by local law.
```
